modules/33acrossBidAdapter.js

```diff
@@ -267,12 +289,57 @@ function _buildBannerORTB(bidRequest) {
     ? _getViewability(element, utils.getWindowTop(), minSize)
     : NON_MEASURABLE;
 
+  const ext = contributeViewability(viewabilityAmount);
+
   return {
     format,
-    ext: contributeViewability(viewabilityAmount)
+    ext
   }
 }
 
+// BUILD REQUESTS: VIDEO
+// eslint-disable-next-line no-unused-vars
+function _buildVideoORTB(bidRequest) {
+  const videoAdUnit = utils.deepAccess(bidRequest, 'mediaTypes.video', {});
+  const videoBidderParams = utils.deepAccess(bidRequest, 'params.video', {});
+
+  const video = {};
+
+  const product = _getProduct(bidRequest);
+
+  // playerSize, assumming an Array with only one Array element
+  // i.e. [[300, 250]] given the Prebid core will validate sizes
+  // set in Ad Unit
+  const sizes = utils.deepAccess(bidRequest, 'mediaTypes.video.playerSize', []);
+  const {w, h} = _getSize(sizes[0]);
+  video.w = w;
+  video.h = h;
+
+  // Placement Inference Rules:
+  // - It could be defined in AdUnit or Bidder Params for 33across
+  // - If no placement is defined then default to 2 (In Banner)
+  // - If product is instream (for instream context) then override placement to 1
+  let placement = videoAdUnit.placement || videoBidderParams.placement;
+  placement = placement || 2;
+
+  if (product === PRODUCT.INSTREAM) {
+    placement = 1;
+  };
+  video.placement = placement;
+
+  // bidfloors
+
+  // startdelay
+  if (product === PRODUCT.INSTREAM) {
+    let startdelay = videoAdUnit.startdelay || videoBidderParams.startdelay;
+    startdelay = parseInt(startdelay) || 0;
+
+    video.startdelay = startdelay;
+  }
+
+  return video;
```

Comment:
how about other variables? `mimes, protocols, api, min, max, playbackmethod`?

Reply:
Sorry, I just updated code to add other video params. Please review [above lines](https://github.com/33Across/Prebid.js/pull/33/files#diff-77276bb121ffb48fb416df947594a23df2e6e7dc6d8560e6ee7319cc3cba56efR330-R334)